Boa.Constrictor.UnitTests/Screenplay/Conditions/Collection/AtLeastOneItemTest.cs

```diff
@@ -0,0 +1,57 @@
+using Boa.Constrictor.Screenplay;
+using FluentAssertions;
+using NUnit.Framework;
+

```

Comment:
For the Condition classes and their unit tests, please rename the `Collection` folder to `Collections` or even `Enumerations`. A plural name would fit established naming conventions better.